src/Jackett.Common/Indexers/Definitions/Abstract/PublicBrazilianIndexerBase.cs

```diff
@@ -0,0 +1,387 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Web;
+using AngleSharp.Dom;
+using CommandLine;
+using Jackett.Common.Extensions;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Newtonsoft.Json.Linq;
+using NLog;
+using static System.Linq.Enumerable;
+using WebClient = Jackett.Common.Utils.Clients.WebClient;
+
+namespace Jackett.Common.Indexers.Definitions.Abstract
+{
+    public abstract class PublicBrazilianIndexerBase : IndexerBase
+    {
+        public PublicBrazilianIndexerBase(IIndexerConfigurationService configService, WebClient wc, Logger l,
+                                          IProtectionService ps, ICacheService cs) : base(
+            configService: configService, client: wc, logger: l, p: ps, cacheService: cs,
+            configData: new ConfigurationData())
+        {
+            webclient.requestDelay = .5;
+        }
+
+        public override string Description =>
+            $"{Name} is a Public Torrent Tracker for Movies and TV Shows dubbed in Brazilian Portuguese";
+
+        public override string Language => "pt-BR";
+        public override string Type => "public";
+        public override TorznabCapabilities TorznabCaps => SetCapabilities();
+
+        private TorznabCapabilities SetCapabilities()
+        {
+            var caps = new TorznabCapabilities
+            {
+                MovieSearchParams = new List<MovieSearchParam> { MovieSearchParam.Q },
+                TvSearchParams = new List<TvSearchParam> { TvSearchParam.Q }
```

Comment:
```suggestion
                TvSearchParams = new List<TvSearchParam> { TvSearchParam.Q, TvSearchParam.Season, TvSearchParam.Ep }
```

Reply:
Sonarr will not work with an indexer whose TV caps is missing the Season and Ep.